<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DynamicVehicleSimTool_UIDialog</class>
 <widget class="QWidget" name="DynamicVehicleSimTool_UIDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_10" stretch="0,1,0">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="margin">
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Simulator Info</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Computer Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="m_computerName">
            <property name="text">
             <string>225.0.0.37</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Port Number</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="m_portNumber"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Simulator Controls</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="m_startStopButton">
            <property name="text">
             <string>Start/Stop</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="m_resetButton">
            <property name="text">
             <string>Reset</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="m_unitsSelector">
            <item>
             <property name="text">
              <string>m -&gt; ft</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>cm -&gt; ft</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>mm -&gt; ft</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>in -&gt; ft</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Geometry Data Mapping</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_12" stretch="1,0,0">
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>431</width>
           <height>144</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="m_choiceListLayout">
          <property name="margin">
           <number>2</number>
          </property>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="m_addButton">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_removeButton">
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_applyButton">
          <property name="text">
           <string>Apply</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>User Constrained Geometry</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="m_userConstrainedGeometrySelector"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Registration Tools</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>SIP Location X, Y, Z (ft)</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLineEdit" name="m_SIPX">
            <property name="text">
             <string>0.0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="m_SIPY">
            <property name="text">
             <string>0.0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="m_SIPZ">
            <property name="text">
             <string>0.0</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>CAD Forward Vector</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="m_CADForwardVector">
            <item>
             <property name="text">
              <string>-X</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>X</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>-Y</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Y</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>-Z</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Z</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>CAD Up Vector</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="m_CADUpVector">
            <item>
             <property name="text">
              <string>-X</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>X</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>-Y</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Y</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>-Z</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Z</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QComboBox" name="m_registrationSelector">
          <item>
           <property name="text">
            <string>Manual</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Choose File...</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_registerButton">
          <property name="text">
           <string>Register</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
